1-Draft/RFCNNNN-Implicit-Line-Continuance.md

```diff
@@ -177,9 +177,49 @@ If the risk is deemed to be too high because of the risk with named unary operat
     Pros: The @ symbol is familiar since it is used for splatting. Also, PSReadline could recognize when you're opting in to use this at the command prompt, only running the command if you hit enter twice (although this wouldn't be used nearly as often at the prompt -- it's really for scripts).
     Cons: Users lose the benefit of not having to use any characters for line continuance.
 
+    Here's another alternative sigil to do the same thing that is a bit more literal:
+
+    ```PowerShell
+    New-ADUser ...
```

Comment:
Good point. Thanks.

Reply:
My current favorite is this alternative -- not the ellipsis specifically, but some sigil to tell the parser to parse the rest of the command in multi-line mode, meaning it would parse parameters/arguments until it encountered a command terminator (e.g. semi-colon, closing brace/bracket, etc.) or a blank line. It's minimal, transitioning to/from that syntax is as simple as adding/deleting the sigil and some newlines, and commands could be wrapped however the scripter wants, it would work in the interactive shell, and it would be completely non-breaking as long as the sigil was chosen carefully.

Reply:
Personally I'm rather fond of using `~` as the continuation token here:

```powershell
Get-ChildItem ~
    C:\
    -Filter *.ps*1
    -File
    -Recurse
```

Reply:
Interesting, when I saw this on my phone without my glasses, I thought the sigil you proposed was `-`, which I liked. I'm less fond of `~`, at least at first glance, but once we decide on an approach to take (assuming we get that far), we can discuss which sigil makes the most sense and share a bunch of options.

Reply:
So, I made the suggestion about a character after trying it out. You guys aren't even thinking about it, nevermind actuallly trying it. The `@` works because it's not valid syntax currently. All those others are already valid input.

```PowerShell
> write-host -{ write-output 'hello' }
- hello

> Write-Host ...
...

> Write-Host ~
~
```

Reply:
> Personally I'm rather fond of using ~ as the continuation token here:
> Get-ChildItem ~
>     C:\
>     -Filter *.ps*1
>     -File
>     -Recurse

In addition to Jaykul's Write-Host example, `~` is a shortcut for the current user's home folder. `Get-ChildItem ~` will return items from that folder.

Reply:
@jaykul actually I'm thinking about this RFC a ton. I just didn't come back to comment again after I replied when I realized neither `-` not `~` would work. In the case of my last comment, it is less about not thinking too hard, more about  responding too quickly to another comment.

Reply:
Not to mention that I rarely if ever use `Write-Host` (which I know you have used a lot), so some of those use cases never occurred to me. Of course that's why we have an RFC for this.